app/webpacker/css/admin/all.scss

```diff
@@ -4,7 +4,7 @@
 @import '~jquery-ui/themes/base/jquery.ui.dialog';
 @import '../shared/textAngular';
 @import '../shared/ng-tags-input.min';
-@import '~select2/select2';
```

Comment:
I'd be happy to remplace select2 by something new (or something to create). Do you have created any issue?

Reply:
I think when we start getting into StimulusReflex we'll need to bring in a different library for selects, yes. This just makes the current `Select2` stuff work with Webpack for now.